docs/modules/reference/pages/provisioning/detectors/BsfDetector.adoc

```diff
@@ -15,43 +15,46 @@ The BSF Detector passes the `map`, `ip_addr`, `svc_name`, and `results` variable
 
 == Configuration and use
 
-.Optional parameters for the BSF service detector
+.Optional parameters for the BSFDetector
 [options="header"]
 [cols="1,3a,1"]
 |===
 | Parameter
 | Description
-| Default
+| Default value
 
 | fileName
-| The path to the file on the OpenNMS server which contains the BSF-compatible script to run.
+| Path to file on OpenNMS server containing BSF-compatible script to run.
 | none
 
 | langClass
-| The short name of the language the script is written in. Choices:
+| Short name of language the script is written in.
```

Comment:
```suggestion
| Short name of the language the script is written in.
```
I think that sometimes removing "the" could cause confusion, particularly for non-native English speakers. Works to remove at the start of the phrase, but not always in the middle.